README.md

```diff
@@ -52,3 +52,48 @@ calculate weighted LD
     if r2 above 0.05 return else skip.
 
 
+# Implementations
+
+## Python
+
+The default implementation of the WeightedLD algorithm is written in Python, and lives in `./WeightedLD.py`. Full usage instructions can be found by running `python WeightedLD.py --help`
+
+In order to run the program, you must have a python environment where the following packages are available:
```

Comment:
There is a conda yml file which we should just mention, i.e. so people dont run python 2 and get a different result

Reply:
I mostly just wrote this section so that there was a rough counterpart to the Rust documentation below, I agree there's a lot of scope to improve it more in the future